content/en/docs/methods/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Methods & Distributions"
```

Comment:
In the meetings, if I recall properly, people prefer `Distributions` more than `Methods`. I'm OK with naming the section "Methods & Distributions" for now, but I'd prefer at least change the path to `/docs/distributions/xxx`.

Reply:
Hey @Bobgy ! Thank you for the comment, I will bring this up for discussion in the next meeting, I am happy with either naming convention. :) 

My understanding was that `kfctl` would be more of a method and e.g. `minikf` more regarded as a distribution, hence the 2 names, but I agree that may not be too stylish and that 1 name would be better than 2.

Reply:
"Methods" is too generic in this context. The term _method_ doesn't imply that it is related to the installation process, just that it is related to _a_ process.

If  "Distritbutions" does not cover all the installation cases, then I would recommend something like "Installing Kubeflow" or "Deploying Kubeflow". The "Installing|Deploying Kubeflow" section could have sub-sections for distributions and methods.

Reply:
@joeliedtke thank you for your comment! You're probably right that "Methods" alone might be too ambiguous.